tables.go

```diff
@@ -284,21 +284,24 @@ func (f fizzer) CreateTable(name string, opts map[string]interface{}, help plush
 
 	if t.Options["timestamps"].(bool) {
 		if !t.HasColumns("created_at", "updated_at") {
-			t.Timestamp("created_at")
-			t.Timestamp("updated_at")
+			if err := t.Timestamp("created_at"); err != nil {
+				return err
+			}
+			if err := t.Timestamp("updated_at"); err != nil {
+				return err
+			}
```

Comment:
Here we want to create only missing columns, that's why we didn't check for the error. `HasColumns` is a OR condition.